_posts/2019-01-23-cockpit-186.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Cockpit 186
+author: Katerina Koukiou
+date: 2019-01-23
+tags: cockpit linux
+slug: cockpit-186
+category: release
+summary: Responsive system page and improvement in services page
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 186.
+
+### Services: Redesign Services page
+
+During usability testing we found out that the service details page was hard to
+understand for administrators with a non-Linux background. This release has
+included some improvements in this field.
```

Comment:
The last sentence is rather fuzzy. Maybe "The Services page got updated to use more neutral and well-known terminology and a simplified layout"? Maybe @garrett has a better idea here. Otherwise this LGTM, thank you!

Reply:
Replaced with the sentence above.